keyboards/kbdcraft/adam0110/info.json

```diff
@@ -0,0 +1,184 @@
+    "bootloader": "tinyuf2",
+    "keyboard_name": "Adam0110",
+    "manufacturer": "KBDCRAFT",
+    "maintainer": "qmk",
+    "url": "https://kbdcraft.store/products/adam",
+    "diode_direction": "ROW2COL",
+    "build": {
+        "lto": false
+    },
+    "rgb_matrix": {
+        "driver": "ws2812",
+        "led_count": 81,
+        "split_count": [64, 17],
+        "max_brightness": 180,
+        "animations": {
+            "alphas_mods": true,
+            "gradient_up_down": true,
+            "gradient_left_right": true,
+            "breathing": true,
+            "band_sat": true,
+            "band_val": true,
+            "band_pinwheel_sat": true,
+            "band_pinwheel_val": true,
+            "band_spiral_sat": true,
+            "band_spiral_val": true,
+            "cycle_all": true,
+            "cycle_left_right": true,
+            "cycle_up_down": true,
+            "rainbow_moving_chevron": true,
+            "cycle_out_in": true,
+            "cycle_out_in_dual": true,
+            "cycle_pinwheel": true,
+            "cycle_spiral": true,
+            "dual_beacon": true,
+            "rainbow_beacon": true,
+            "rainbow_pinwheels": true,
+            "raindrops": true,
+            "jellybean_raindrops": true,
+            "hue_breathing": true,
+            "hue_pendulum": true,
+            "hue_wave": true,
+            "pixel_rain": true,
+            "solid_reactive_simple": true,
+            "solid_reactive": true,
+            "solid_reactive_wide": true,
+            "solid_reactive_multiwide": true,
+            "solid_reactive_cross": true,
+            "solid_reactive_multicross": true,
+            "solid_reactive_nexus": true,
+            "solid_reactive_multinexus": true,
+            "splash": true,
+            "multisplash": true,
+            "solid_splash": true,
+            "solid_multisplash": true
+        }
+    },
+    "ws2812": {
+        "driver": "pwm",
+        "pin": "B13"
+    },
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": true,
+        "extrakey": true,
+        "mousekey": false,
+        "nkro": false,
+        "rgb_matrix": true
+    },
+    "usb": {
+        "device_version": "0.0.0",
+        "vid": "0x586A",
+        "pid": "0xB002"
+    },
+    "split": {
+        "enabled": true,
+        "soft_serial_pin": "A9",
+        "transport": {
+            "protocol": "serial_usart",
+            "sync_matrix_state": true,
+            "watchdog": true
+        },
+        "usb_detect": {
+            "enabled": true,
+            "timeout": 5000,
+            "polling_interval": 20
+        }
+    },
+    "matrix_pins": {
+        "cols": [
+            "C10",
+            "C11",
+            "C12",
+            "A15",
+            "B6",
+            "D2",
+            "B3",
+            "B4",
+            "B5",
+            "B7",
+            "A0",
+            "A1",
+            "A2",
+            "A3"
+        ],
```

Comment:
Please have the `cols` pins be laid out in a single line instead of being separated by spaces.